infra/backup.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+#
+# SSH into our Clerk EC2 instance
+# Backup the Postgres DB and upload to S3
+#
+set -e
+HOST='13.55.250.149'
+TIME=$(date "+%s")
+DATABASE_NAME=clerk
+S3_BUCKET=s3://anika-database-backups
+BACKUP_FILE="postgres_${DATABASE_NAME}_${TIME}.sql.gz"
+S3_PATH="$S3_BUCKET/$BACKUP_FILE"
+
+if [[ -z "$CLERK_PRIVATE_SSH_KEY" ]]
+then
+    echo -e "\n>>> Error: Clerk private key not found in CLERK_PRIVATE_SSH_KEY."
```

Comment:
What is the `-e` flag for? Does this mean that if the echo command fails we should exit the script?

Reply:
`-e` means print `\n` as a new line

```enable interpretation of backslash escapes```

Reply:
distinct from `set -e` which causes the script to halt on errors